src/installation/installation.md

```diff
@@ -1,32 +1,106 @@
 
-Epinio is a Kubernetes application and thus needs a running Kubernetes
-cluster for installation.
+You can disable the installation of `Kubed` and `MinIO` by changing the settings as described in the respective sections.
 
-See [system requirements](system_requirements.md) about the Kubernetes
-environment Epinio expects to run on.
 
-## Installation Methods
+## Prerequisites
 
-There are at least 2 ways to install Epinio on a Kubernetes cluster. Which one
-you choose depends on the target environment and the amount of customization
-you want over the default Epinio installation.
+See [system requirements](system_requirements.md) for a detailed list of external components your
+Kubernetes cluster needs to have before you install Epinio.
 
-1. [Install Epinio and automatically install dependencies](install_epinio_auto.md) - Start here if you are new to Epinio
-2. [Install Epinio and manually install components](install_epinio_manual.md) - Full control over installation, mostly for production setups
+> **IMPORTANT:** Some of the namespaces of the components are hardcoded in the Epinio
+code and thus are important to be the same as described here. In the future this
+may be configurable on the Epinio Helm chart.
 
-Epinio is not just one application running on your cluster. It depends on other Kubernetes components for some of its functionality. The 2 different installation methods above, provide different level of configurability on how you get those dependencies installed on your cluster. If you are just starting out with a fresh cluster and you don't have an opinion on how things should be installed, then the first method is the best for you.
+## Installation
+### Ingress Controller
 
-If you've done your due diligence and you are now ready to deploy Epinio on your production clusters, maybe your Dev Ops team wants to fully control what is being installed an how. In that case, the second installation option may be better.
+Epinio creates Ingress resources for the API server, the applications and depending
+on your setup, the internal container registry. Those resources won't work unless
+an Ingress controller is running on your cluster.
+
+If you don't have an Ingress controller already running, you can install Traefik with:
+
+```
+$ kubectl create namespace traefik
+$ export LOAD_BALANCER_IP=$(LOAD_BALANCER_IP:-) # Set this to the IP of your load balancer if you know that
+$ helm install traefik --namespace traefik "https://helm.traefik.io/traefik/traefik-10.3.4.tgz" \
+		--set globalArguments='' \
+		--set-string ports.web.redirectTo=websecure \
+		--set-string ingressClass.enabled=true \
+		--set-string ingressClass.isDefaultClass=true \
+		--set-string service.spec.loadBalancerIP=$LOAD_BALANCER_IP
+```
+
+### Cert Manager
+
+Epinio needs [cert-manager](https://cert-manager.io/) in order to create TLS
+certificates for the various Ingresses (see "Ingress controller" above).
+
```

Comment:
I'm not sure about this level of details but if cert-manager is already installed in a namespace different than cert-manager, you have to set it via `.Values.certManagerNamespace`, otherwise installation will fail.

Reply:
Added a `warning` quote.